src/app/search-bar/search-bar.component.html

```diff
@@ -11,15 +11,15 @@
       #searchInput
       placeholder="Search in our amazing collection of items"
     />
-    <button class="__button--clear" type="reset" (click)="resetForm()">
+    <button class="SearchBar__clear" type="reset" (click)="resetForm()">
```

Comment:
You're loosing the context of the button here.
`SearchBar__button__clear` and `SearchBar__button__submit` look clearer to me.